Modules/Server/Sources/PocketCastsServer/Public/API/ApiServerHandler.swift

```diff
@@ -5,6 +5,11 @@ import SwiftProtobuf
 
 public class ApiServerHandler {
     public static let shared = ApiServerHandler()
+    internal let urlSession: URLSession
```

Comment:
I wonder if we change this to `private`?

Reply:
Same idea mentioned [here](https://github.com/Automattic/pocket-casts-ios/pull/388/files/784a9d8e24e038190c824909660d2c06b00fe842#r994747790).

Happy to drop the identifier

Reply:
Oh, I haven't realized that this is not `private` because it's used in an extension. That seems a wrong reason to have this `internal`. 😬

But refactoring it is out of the scope of this task.